libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -115,9 +115,9 @@ class AC_AttitudeControl {
 
     // get the yaw angular velocity limit in radians/s
     float get_ang_vel_yaw_max_rads() const { return radians(_ang_vel_yaw_max); }
-    
-    // get the yaw slew limit
-    float get_slew_yaw_cds() const { return _slew_yaw; }
+
+    // get the yaw rate limit in deg/s
+    float get_max_yaw_rate() const;
```

Comment:
can we change this to:
get_ang_vel_yaw_max_degs

Reply:
changed to `get_slew_ang_vel_yaw_max_degs`